apps/namegraph.dev/lib/utils.ts

```diff
@@ -1,6 +1,126 @@
 import { clsx, type ClassValue } from "clsx";
 import { twMerge } from "tailwind-merge";
+import {
+  NameGraphGroupedByCategoryResponse,
+  NameGraphGroupingCategory,
+  NameGraphSuggestion,
+} from "@namehash/namegraph-sdk/utils";
+import { createNameGraphClient } from "@namehash/namegraph-sdk";
 
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+interface LabelAndModeQueryParams {
+  label: string;
+  mode: "full";
+}
+
+export interface NameGraphRelatedCollection {
```

Comment:
Is there a reason this isn't in the SDK? Is it a part of the API?

Reply:
This was actually a duplicate, nice you find it